### PR TITLE
iterate preview and variant

### PR DESCRIPTION
**Pull request by the-daniel-rothig**
### Context

https://manage-courses-prototype.herokuapp.com/history/preview-course

### Changes proposed in this pull request

pull in new version of view component

### Guidance to review

It's Friday.

### CODE REVIEW

File: src/ui/ManageCoursesUi.csproj [hunk 1]
```diff
@@ -32,10 +32,10 @@
     <PackageReference Include="GovUk.Education.ManageCourses.ApiClient" Version="0.9.2.*" />
   </ItemGroup>
   <ItemGroup Condition="Exists('$(ProjectDir)dev-sc-shared.targets')==false">    
-    <PackageReference Include="GovUk.Education.SearchAndCompare.Ui.Shared" Version="0.2.0" />
+    <PackageReference Include="GovUk.Education.SearchAndCompare.Ui.Shared" Version="0.3.0" />
   </ItemGroup>
   <Target Name="Webpack" BeforeTargets="AfterBuild">
-    <Exec Command="npm install" />
-    <Exec Command="npm run build" />
+    <!-- <Exec Command="npm install" />
```
Comment (dankmitchell) [inline src/ui/ManageCoursesUi.csproj]: put this back

Comment (defong) [reply src/ui/ManageCoursesUi.csproj]: hee hee

**Comment by defong** [comment]
- [ ] Missing Accrediting by xxx?
- [ ]  Missing Border fattest for the table header for available training locations

mc-preview-1

**Comment by defong** [comment]
Clarification needed, copy text is different

image

**Comment by the-daniel-rothig** [comment]
@defong todo done, copy is separate post-it, see whiteboard

File: src/ui/Assets/Javascript/back-link.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ BackLink.prototype.init = function () {
       var $link = document.createElement("a");
       $link.setAttribute("href", "javascript:history.back()");
       $link.setAttribute("class", "govuk-back-link")
-      $link.textContent = "Back to search results";
+      $link.textContent = "Back to course";
```
Comment (defong) [inline src/ui/Assets/Javascript/back-link.js]: back

File: src/ui/Assets/Styles/site.scss [hunk 3]
```diff
@@ -50,6 +50,12 @@ $govuk-global-styles: true;
   border: 4px solid govuk-colour("red");
 }
 
+// custom table header border tickness
+.govuk-table__header {
+  border-bottom-width: 2px;
+  border-bottom-color: govuk-color("black");
```
Comment (defong) [inline src/ui/Assets/Styles/site.scss]: image

File: src/ui/Assets/Javascript/back-link.js (hunk 2 above)
Comment (the-daniel-rothig) [reply src/ui/Assets/Javascript/back-link.js]: pre-existing issue, carded up

File: src/ui/Assets/Styles/site.scss (hunk 3 above)
Comment (the-daniel-rothig) [reply src/ui/Assets/Styles/site.scss]: that's not how you spell colour is it 😬

**Comment by the-daniel-rothig** [comment]
@defong oops another css class typo. fixed now

**Pull request merged**